test/e2e/assets/chart/empty.go

```diff
@@ -0,0 +1 @@
+package chart
```

Comment:
nit: a `doc.go` file with comments that explain the purpose of the package might be nicer.